tutor/src/screens/task/step/card.js

```diff
@@ -25,6 +25,7 @@ const LoadingCard = styled(InnerStepCard)`
   min-width: 960px;
   padding: 2rem;
 `;
+LoadingCard.displayName = 'LoadingCard';
```

Comment:
Maybe displayName is actually not displayed? Hmm odd name choice :)

Reply:
it's only used  in the React dev tools and in jest/enzyme debugging